src/Pair/toPairs.js

```diff
@@ -1,21 +1,20 @@
 /** @license ISC License (c) copyright 2017 original and current authors */
 /** @author Ian Hofmann-Hicks (evil) */
 
-const List = require('../core/List')
 const Pair = require('../core/Pair')
 const isObject = require('../core/isObject')
 
-// toPairs : Object -> List (Pair String a)
+// toPairs : Object -> Array (Pair String a)
```

Comment:
`[ (Pair String a) ]` or `Array (Pair String a)`?

Reply:
I am good with how you have it, reads better IMO

Reply:
I agree

Reply:
I agree